src/phases/execution/MultiDeletes.yml

```diff
@@ -7,34 +7,67 @@ Description: >
   The mass deletion namespace is __internalBatchedDeletesTesting.Collection0
   The concurrent writes namespace is __internalBatchedDeletesTesting.concurrentWritesColl
 
+Keywords:
+- RunCommand
+- Loader
+- LoggingActor
+- CrudActor
+- insert
+- delete
+- batch
+- deleteMany
+- latency
+
 GlobalDefaults:
   # Configurable parameters.
   DocumentCount: &DocumentCount {^Parameter: {Name: "DocumentCount", Default: 100_000_000}}
+  # Number of documents to batch into an insert. This should be configurable because large docs may hit a memory limit if the batch size is fixed.
+  InsertBatchSize: &InsertBatchSize {^Parameter: {Name: "InsertBatchSize", Default: 1_000}}
   # The size of the documents inserted and deleted into collections during this test.
-  DocumentSizePaddingBytes: &DocumentSizePaddingBytes {^Parameter: {Name: "DocumentSizePaddingBytes", Default: 0}}
+  TargetDocumentSizeBytes: &TargetDocumentSizeBytes {^Parameter: {Name: "TargetDocumentSizeBytes", Default: 1_024}}
+  # Compute document padding based on target size and field sizes
+  DocumentPaddingSizeBytes: &DocumentPaddingSizeBytes
+    ^NumExpr:
+      withExpression: "target_size - id_length - (field_count * (field_length+value_length))"
+      andValues: {target_size: *TargetDocumentSizeBytes, id_length: 22, field_count: 7, field_length: 8, value_length: &ValueLength 6}
```

Comment:
Thanks for this! On the 1kB workload it generates documents of 1038 bytes, which is acceptably close to our target. And in the future we'll be able to easily scale up and down with the number of secondary indexes.

Reply:
Seems like I forgot to also take into account the size of the "padding" field itself. Fixed it!